docs/01-overview/application-connectivity/README.md

```diff
@@ -2,138 +2,4 @@
-
-- Perform some simple and more advanced tasks:
-
-  - [Pass the access token in the request header](../../04-operation-guides/operations/ac-01-pass-access-token-in-request-header.md)
-  - [Create a new Application](../../03-tutorials/00-application-connectivity/ac-01-create-application.md)
-  - [Register a service](../../03-tutorials/00-application-connectivity/ac-03-register-manage-services.md)
-  - [Register a secured API](../../03-tutorials/00-application-connectivity/ac-04-register-secured-api.md)
-  - [Call a registered external service from Kyma](../../03-tutorials/00-application-connectivity/ac-05-call-registered-service-from-kyma.md)
-  - [Disable TLS certificate verification](../../03-tutorials/00-application-connectivity/ac-11-disable-tls-certificate-verification.md)
-  - [Enable Kyma with Runtime Agent](../../04-operation-guides/operations/ra-01-enable-kyma-with-runtime-agent.md)
-  - [Establish a secure connection with Compass](../../03-tutorials/00-application-connectivity/ra-01-establish-secure-connection-with-compass.md)
-  - [Maintain a secure connection with Compass](../../03-tutorials/00-application-connectivity/ra-02-maintain-secure-connection-with-compass.md)
-  - [Revoke a client certificate (RA)](../../03-tutorials/00-application-connectivity/ra-03-revoke-client-certificate.md)
-  - [Configure Runtime Agent with Compass](../../03-tutorials/00-application-connectivity/ra-04-configure-runtime-agent-with-compass.md)
-  - [Reconnect Runtime Agent with Compass](../../03-tutorials/00-application-connectivity/ra-05-reconnect-runtime-agent-with-compass.md)
-
-- Analyze Application Connectivity specification and configuration files:
-
-  - [Application](../../05-technical-reference/00-custom-resources/ac-01-application.md) custom resource (CR)
-  - [Application Connector chart](../../05-technical-reference/00-configuration-parameters/ac-01-application-connector-chart.md)
-  - [Compass Connection](../../05-technical-reference/00-custom-resources/ra-01-compassconnection.md) custom resource (CR)
-  - [Connection with Compass](../../05-technical-reference/00-configuration-parameters/ra-01-connection-with-compass.md) 
-
-- Understand technicalities behind the Application Connectivity implementation:
-
-  - [Application Connector components](../../05-technical-reference/00-architecture/ac-01-application-connector-components.md)
-  - [Application Gateway workflow](../../05-technical-reference/00-architecture/ac-03-application-gateway.md)
-  - [Application Gateway details](../../05-technical-reference/ac-01-application-gateway-details.md)
-  - [Runtime Agent workflow](../../05-technical-reference/00-architecture/ra-01-runtime-agent-workflow.md)
-  - [Configuring the Runtime](../../05-technical-reference/ra-01-configuring-runtime.md)
+This content has been moved to: [Application Connector module](https://kyma-project.io/#/application-connector-manager/user/README).
```

Comment:
Should we leave this doc for the sake of this one line? I can see that Serverless, Telemetry, and Service Management also left their READMEs in the `01-overview` but I'm not sure if there is a need for that. Or we should leave it or we should leave those for now and clean up after the modularization is complete?

Reply:
I'd keep it and clean up later, as you suggested.